src/main/java/io/quarkus/bot/PullRequestCommandHandler.java

```diff
@@ -0,0 +1,69 @@
+package io.quarkus.bot;
+
+import io.quarkiverse.githubapp.event.IssueComment;
+import io.quarkus.bot.command.Command;
+import io.quarkus.bot.config.QuarkusBotConfig;
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHOrganization;
+import org.kohsuke.github.GHTeam;
+import org.kohsuke.github.GHUser;
+
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PullRequestCommandHandler {
+
+    private static final Logger LOG = Logger.getLogger(PullRequestCommandHandler.class);
+
+    private static final String QUARKUS_BOT_NAME = "quarkus-bot";
+    private static final String QUARKUS_TEAM_MEMBERS = "quarkus-push";
+    private static final Pattern QUARKUS_BOT_MENTION = Pattern.compile("^@(?:quarkus-?)?bot\\s+([a-z _\\-]+)");
+
+    @Inject
+    QuarkusBotConfig quarkusBotConfig;
+
+    @Inject
+    Instance<Command> commands;
+
+    public void onComment(@IssueComment.Created @IssueComment.Edited GHEventPayload.IssueComment commentPayload)
+            throws IOException {
+        GHUser user = commentPayload.getComment().getUser();
+        GHIssue issue = commentPayload.getIssue();
+
+        if (issue.isPullRequest()
+                && !QUARKUS_BOT_NAME.equals(user.getName())) {
+            Optional<Command> ghIssueConsumer = extractCommand(commentPayload.getComment().getBody());
+            if (ghIssueConsumer.isPresent() && isQuarkusTeamMember(user, commentPayload.getOrganization())) {
```

Comment:
Yeah good thing to check the permission second to limit the number of API calls.